ht.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
Had a small clarification: Is this html file just the compilation of all HTML code which you have written to describe the avatars?

Reply:
fixed - d024da009973ecafa0fe044faac36624c2778267

Reply:
Got it Sukanya!!You can merge it.